plugins/mcreator-core/procedures/world_use_bonemeal.json

```diff
@@ -14,6 +14,12 @@
       "type": "input_value",
       "name": "z",
       "check": "Number"
+    },
+    {

```

Comment:
This one also seems ok on client-side, just asking what was the reason here (asking for own understanding of how it works)?

Reply:
OKay, I've mistaken this particular one, others seem to block client side properly.